src/main/java/org/jalasoft/moi/controller/endpoints/LanguageController.java

```diff
@@ -44,7 +45,7 @@
      */
     @RequestMapping(method = RequestMethod.PUT, path = "/q")
     public String processInput(@RequestParam(value = "userInput") String userInput,
-                               @RequestParam(value = "pid") Long pid) {
+                               @RequestParam(value = "pid") Long pid) throws ResultException, InputParametersException {
         InputParameters answer = new Answer();
```

Comment:
Here the same case, I believe you should not throw an exception in the signature of the method.